tuxemon/formula.py

```diff
@@ -202,3 +203,15 @@ def simple_overfeed(
 ) -> int:
     speed = target.speed // 2
     return speed
+
+
+def escape(level_user: int, level_target: int, run: int) -> int:
+    if run == 0:
```

Comment:
Could just be attempts = run, but why not just rename the "run" arg to "attempts"?

Reply:
Thanks, I didn't notice, renamed.